### PR TITLE
Feature/widgets-structure-styles

### PR DESCRIPTION
**Pull request by hectoruch**
## Overview

This PR include:
- Improve structure styles.
- Fix footer.
- Include countries footer.
- Include select´s styles and functions on the header.
- Break the margins on country page.

## Demo

- After:
pasted image at 2017_09_26 05_59 pm

- Before:
pasted image at 2017_09_26 06_00 pm

- Select
selectheader

- Footer
image

## Notes

(No notes)

## Testing

- Visit the country page and test it.

### CODE REVIEW

**Comment by hectoruch** [comment]
Merge -> https://github.com/Vizzuality/gfw/pull/3051

**Comment by hectoruch** [comment]
Merge -> https://github.com/Vizzuality/gfw/pull/3052

**Comment by hectoruch** [comment]
Merge -> https://github.com/Vizzuality/gfw/pull/3053

**Comment by hectoruch** [comment]
Merge -> https://github.com/Vizzuality/gfw/pull/3056

File: app/javascript/pages/country/components/widget-areas-most-cover-gain/widget-areas-most-cover-gain.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  endYear: 2015,
+  thresh: 30
+});
+
+import {
+  getTreeCoverGainRegion
+} from '../../../../services/tree-gain';
+
+import {
+  getCountryRegions
```
Comment (pjosh) [inline app/javascript/pages/country/components/widget-areas-most-cover-gain/widget-areas-most-cover-gain.js]: You don't need to do that! Country regions is already on redux.

File: app/javascript/pages/country/components/widget-areas-most-cover-gain/widget-areas-most-cover-gain.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+const WidgetAreasMostCoverGainContainer = (props) => {
+  const setInitialData = (props) => {
+    getTreeCoverGainRegion(
+      props.iso,
+      {minYear: props.startYear, maxYear: props.endYear},
+      props.thresh
+    )
+    .then((treeCoverGainByRegion) => {
+      getCountryRegions(props.iso)
```
Comment (pjosh) [inline app/javascript/pages/country/components/widget-areas-most-cover-gain/widget-areas-most-cover-gain.js]: You don't need to do that! Country regions is already on redux.

File: app/javascript/pages/country/components/widget-tree-cover-loss-areas/widget-tree-cover-loss-areas.js [hunk 3]
```diff
@@ -14,25 +14,43 @@ const mapStateToProps = state => ({
...
+  endYear: 2015,
+  thresh: 30
 });
 
+import {
+  getTreeLossByRegion
+} from '../../../../services/tree-loss';
+
+import {
+  getCountryRegions
```
Comment (pjosh) [inline app/javascript/pages/country/components/widget-tree-cover-loss-areas/widget-tree-cover-loss-areas.js]: You don't need to do that! Country regions is already on redux.

File: app/javascript/pages/country/components/widget-tree-cover-loss-areas/widget-tree-cover-loss-areas.js [hunk 4]
```diff
@@ -14,25 +14,43 @@ const mapStateToProps = state => ({
...
-      { name: 'Mato Grosso', value: 203, color: '#cecb65' },
-      { name: 'Other Districts', value: 3000, color: '#e9e9ea' }
-    ]);
+    getTreeLossByRegion(
+      props.iso,
+      {minYear: props.startYear, maxYear: props.endYear},
+      props.thresh
+    )
+    .then((treeLossByRegion) => {
+      getCountryRegions(props.iso)
```
Comment (pjosh) [inline app/javascript/pages/country/components/widget-tree-cover-loss-areas/widget-tree-cover-loss-areas.js]: You don't need to do that! Country regions is already on redux.

File: app/javascript/pages/country/components/widget-tree-located/widget-tree-located.js [hunk 5]
```diff
@@ -8,6 +9,15 @@ export { initialState } from './widget-tree-located-reducers';
 export { default as reducers } from './widget-tree-located-reducers';
 export { default as actions } from './widget-tree-located-actions';
 
+import {
+  getTotalCover,
+  getTotalCoverRegions
+} from '../../../../services/tree-cover';
+
+import {
+  getCountryRegions
```
Comment (pjosh) [inline app/javascript/pages/country/components/widget-tree-located/widget-tree-located.js]: You don't need to do that! Country regions is already on redux.

File: app/javascript/pages/country/components/widget-tree-located/widget-tree-located.js [hunk 6]
```diff
@@ -16,60 +26,29 @@ const mapStateToProps = state => ({
...
-        name: 'Mato Grosso',
-        value: 203000,
-        percent: 5
-      }
-    ]);
+    getTotalCover(props.iso, props.countryRegion)
+    .then((totalCoverResponse) => {
+      getTotalCoverRegions(props.iso)
+      .then((totalCoverRegions) => {
+        getCountryRegions(props.iso)
```
Comment (pjosh) [inline app/javascript/pages/country/components/widget-tree-located/widget-tree-located.js]: You don't need to do that! Country regions is already on redux.

File: package.json [hunk 7]
```diff
@@ -77,12 +77,16 @@
     "react-router-config": "^1.0.0-beta.4",
     "react-router-dom": "^4.2.2",
     "react-scroll": "^1.5.4",
+    "react-select-me": "^0.9.0",
+    "react-select2": "^4.0.3",
+    "react-virtualized-select": "^3.1.0",
```
Comment (pjosh) [inline package.json]: 3 selects?

**Pull request merged**